### PR TITLE
Feature ddata

### PR DESCRIPTION
**Pull request by essgje**
# \<Description\>

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

**Review by tommigitgaur — APPROVED** [review]
This is awsome

File: apps/skilavottord/ws/src/app/api.graphql [hunk 1]
```diff
@@ -2,22 +2,61 @@
...
+  name: String!
+  mobile: String
+  cars: [Car!]
 }
 
 type Carowner {
   nationalId: ID!
   name: String!
   mobile: String
+<<<<<<< HEAD
```
Comment (pshomov) [inline apps/skilavottord/ws/src/app/api.graphql:35]: It seems you have done some merge at some point which did not go well and now you need to clean it up before it is usable again.

File: apps/skilavottord/ws/src/app/app.module.ts [hunk 2]
```diff
@@ -1,8 +1,16 @@
 import { Module } from '@nestjs/common'
 import { GraphQLModule } from '@nestjs/graphql'
 
+<<<<<<< HEAD
```
Comment (pshomov) [inline apps/skilavottord/ws/src/app/app.module.ts:4]: Same here and other places

**Review by pshomov — CHANGES_REQUESTED** [review]
You should cleanup the source and then we can see where this stands in regards to everything else

**Comment by essgje** [comment]
chancel pull-request make new when fixed

**Pull request closed without merging**